apps/docs/content/components/LogoSuite/react.mdx

```diff
@@ -227,6 +226,39 @@ A heading element must be present as it serves as a navigation hook for assistiv
 </LogoSuite>
 ```
 
+### Heading with explicit level
```

Comment:
Nit... can we use 'alternative' over explicit like we do in [River](https://primer-685dfc3147-26139705.drafts.github.io/brand/components/River/react#alternative-heading-levels). Bit softer.

```suggestion
### Alternative heading levels
```